InvenTree/common/models.py

```diff
@@ -1775,7 +1775,7 @@ def save(self, *args, **kwargs):
         'RETURNORDER_REFERENCE_PATTERN': {
             'name': _('Return Order Reference Pattern'),
             'description': _(
-                'Required pattern for generating Return Order reference field'
+                'Required pattern for generating Return Order referenfce field'
```

Comment:
This might be a typo? @SchrodingersGat

Reply:
@wolflu05 oops, yeah good catch

Reply:
https://github.com/inventree/InvenTree/pull/6742